nengo_viz/static/viz_tooltips.js

```diff
@@ -2,25 +2,21 @@ VIZ.tooltips = {};
 
 VIZ.tooltips.tooltip = function($parent, content, placement, icon) {
     if (typeof(placement) === 'undefined') { placement = "bottom"; }
-    if (typeof(icon) === 'undefined') { icon = "glyphicon-question-sign"; }
 
     var $tooltip = $('<a href="#" data-toggle="tooltip" ' +
                      'data-placement="' + placement + '" ' + 'title="' +
                      content + '"/>');
-    $tooltip.append('<span class="glyphicon ' + icon + '" ' +
-                    'aria-hidden="true"/>').appendTo($parent);
+    $tooltip.append('<sup>?</>').appendTo($parent);
```

Comment:
Should this (and below) be `<sup>?</sup>`? Or does jquery somehow figure out that ending tag automatically?